lexers/c/testdata/csharpaspx_script_language.aspx

```diff
@@ -0,0 +1,3 @@
+<script language="C#" type="text/C#" runat="server">
+    // stuff goes here...
+}
```

Comment:
Same question as in the other PR. Is this curly bracket correct?

Reply:
Fixed it. Thanks.